tools/scxtop/src/lib.rs

```diff
@@ -57,7 +59,7 @@ pub const TRACE_FILE_PREFIX: &str = "scxtop_trace";
 pub const STATS_SOCKET_PATH: &str = "/var/run/scx/root/stats";
 pub const LICENSE: &str = "Copyright (c) Meta Platforms, Inc. and affiliates.
 
-This software may be used and distributed according to the terms of the 
+This software may be used and distributed according to the terms of the
```

Comment:
Double check this, I think the extra space is there on purpose. It could also be that the string formatting for the license in the help section could use some better formatting.
<img width="1129" alt="image" src="https://github.com/user-attachments/assets/ac9726ec-88d0-4387-829b-92ef8e81ddd3" />